### PR TITLE
Modinfo core collection.

### PR DESCRIPTION
**Pull request by jsvob**
https://github.com/RedHatInsights/insights-core/issues/1793

### CODE REVIEW

File: insights/combiners/modinfo.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+@combiner(ModInfoEach)
+class ModInfo:
+    """
+    Combiner for accessing all the modinfo outputs.
+
+    Examples:
+        >>> type(modinfo_obj)
+        <class 'insights.combiners.modinfo.ModInfo'>
```
Comment (xiangce) [inline insights/combiners/modinfo.py]: In the test of python2, here `<type instance>` is expected, to the correct expected, better inherit the combiner class from `object` explicitly, e.g `class ModInfo(object):` Or `class ModInfo(dict):` just like the comment above.

Comment (matysek) [reply insights/combiners/modinfo.py]: @xiangce do we still need to retain python2 compatibility for insights-core? I though we are py3 only here.

Comment (jsvob) [reply insights/combiners/modinfo.py]: @matysek : @xiangce has kindly explained on today's mtg that insights-core will be run in environments with python 2.6 and 2.7, so we need to have that compatibility. I didn't realize it before.

**Comment by xiangce** [comment]
@jsvob - This PR is ready to merge after the following flake8 issues being fixed:

```
+ flake8
./insights/combiners/tests/test_modinfo.py:15:9: F841 local variable 'x' is assigned to but never used
```

**Pull request merged**